docs/sites.yml

```diff
@@ -8334,3 +8334,14 @@
   built_by: Peter Brady
   built_by_url: https://www.peterbrady.co.uk
   featured: false
+- title: Salexa - Estetica Venezolana
+  url: https://peluqueriavenezolana.cl/
+  main_url: https://peluqueriavenezolana.cl/
+  source_url: https://github.com/enbonnet/salexa-front
+  description: >
+    Sitio de una estetica o peluqueria de venezolanos en Chile
```

Comment:
Please use english description for our showcase

Reply:
Already update this, thanks for the correction